src/Dialog.tsx

```diff
@@ -111,6 +127,37 @@ export default class Dialog extends React.Component<IDialogPropTypes, any> {
       this.removeScrollingEffect();
     }
   }
+  // 检查边界限定
```

Comment:
Is that possible move all the drag logic into `Draggable.jsx`?

Reply:
I‘ve tried，but i found it hard to get the moved element dom, because I drag the title dom but move the dialog dom.I cant find an appropriate way to get the moved element dom.That's why I leave the checkBorder in Dialog.tsx.

Reply:
Maybe i should provide 2 wrapperedComopent into Dialog, one to drag, another to move; Is this OK?

Reply:
In checkedBorder, content domNode is uesed to get the width of modal, and dialog domNode is used to get the initX (marginLeft) and initY (marginY). if move this logic to Draggable.tsx, I dont think it would be more clear. Maybe there's another way to check the border that I dont konw, please give me a hint.